service/storage_service.cc

```diff
@@ -2462,6 +2462,18 @@ future<node_ops_cmd_response> storage_service::node_ops_cmd_handler(gms::inet_ad
                 node_ops_cmd_response resp(ok, ops_uuids);
                 slogger.debug("node_ops_cmd_handler: Got query_pending_ops request from {}, pending_ops={}", coordinator, ops_uuids);
                 return resp;
+            } else if (req.cmd == node_ops_cmd::repair_updater) {
+                slogger.debug("repair[{}]: Got repair_updater request from {}", ops_uuid, coordinator);
+                ss.db().invoke_on_all([coordinator, ops_uuid, tables = req.repair_tables] (database &db) {
+                    for (const auto& table_id : tables) {
+                        auto& table = db.find_column_family(table_id);
+                        table.update_off_strategy_trigger();
+                        slogger.info("repair[{}]: Updated off_strategy_trigger for table {}.{} by node {}",
```

Comment:
will not this cause a bloat in the log during intense repair activity? perhaps we should log this only on shard 0

given that we chose info level for this, better to use terms that users will understand.

Reply:
The log happens once per 30 seconds. Perhaps it is best to make it debug.